.github/workflows/ci.yml

```diff
@@ -76,12 +76,13 @@ jobs:
           go-version: 1.17.x
       - name: make test
         run: make test
-  e2e:
-    name: e2e test
+  kubernetes:
+    name: Kubernetes
     needs: build
     runs-on: ubuntu-18.04
-    env:
-      ARTIFACTS_PATH: ${{ github.workspace }}/karmada-e2e-logs/
+    strategy:
+      matrix:
+        k8s: [v1.21.2, v1.22.0, v1.23.0]
```

Comment:
the minimum kubernetes version in compat matrix is 1.15, should we add it as well ?

Reply:
Yes, plan to do so, the current three k8s versions test ok, the others I will test and add later.

Reply:
That depends, we've done testing manually against v1.15, but it will be complicated to run it on CI, because if we can't select v1.15 as host cluster (no CRD@v1 support). 

So, my opinion is, to support as much as we can for now with limited effort.